.example.env

```diff
@@ -0,0 +1,14 @@
+DBLINK = postgres://users_5p3h_user:gNzkEQhb4BiBs4Ivd19gPkBDjTLFF4kc@dpg-cf9ngp4gqg47p3vrnh00-a.oregon-postgres.render.com/users_5p3h
```

Comment:
don't share this file.
with the .env information.

Reply:
ooh yeah you are sure and  I already fix it thank you